include/envoy/router/rds.h

```diff
@@ -22,6 +23,12 @@ class RouteConfigProvider {
    */
   virtual Router::ConfigConstSharedPtr config() PURE;
 
+  /**
+   * @return envoy::api::v2::RouteConfiguration the underlying RouteConfiguration object associated
+   * with this provider.
+   */
+  virtual envoy::api::v2::RouteConfiguration configAsProto() const PURE;
```

Comment:
I think you can return a `const envoy::api::v2::RouteConfiguration&` here. You can use `default_instance()` in places where you need a reference to a default configuration.

Reply:
👍